content/docs/cucumber/configuration.md

```diff
@@ -231,7 +231,48 @@ This is just a convention though; Cucumber will pick them up from any file{{% te
 # Profiles
 
 {{% block "java" %}}
-Profiles are not available in Java.
+Profiles are not available in Java.  However, it is possible to set configuration options using [Maven profiles](https://maven.apache.org/guides/introduction/introduction-to-profiles.html).
```

Comment:
Should we be extra clear here and say that the Cucumber feature of profiles is not available on Cucumber-jvm?

Reply:
Fixed!